aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/model/CognitoAuthorizerClaims.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.amazonaws.serverless.proxy.internal.model;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * This object represents the claims property in the authorizer context of a request. The claims object is normally populated
+ * by a Cognito User Pool authorizer and contains the following fields:
+ * <pre>
+ * "claims": {
+ *     "sub": "42df3b02-29f1-4779-a3e5-eff92ff280b2",
+ *     "aud": "2k3no2j1rjjbqaskc4bk0ub29b",
+ *     "email_verified": "true",
+ *     "token_use": "id",
+ *     "auth_time": "1492467169",
+ *     "iss": "https://cognito-idp.us-east-2.amazonaws.com/us-east-2_Adx5ZHePg",
+ *     "cognito:username": "sapessi",
+ *     "exp": "Mon Apr 17 23:12:49 UTC 2017",
+ *     "iat": "Mon Apr 17 22:12:49 UTC 2017",
+ *     "email": "bulianis@amazon.com"
+ * }
+ * </pre>
+ */
+public class CognitoAuthorizerClaims {
+
+    //-------------------------------------------------------------
+    // Constants
+    //-------------------------------------------------------------
+
+    // Mon Apr 17 23:12:49 UTC 2017
+    static final DateTimeFormatter TOKEN_DATE_FORMATTER = DateTimeFormatter.ofPattern("EEE MMM dd HH:mm:ss zzz yyyy");
+
+
+    //-------------------------------------------------------------
+    // Variables - Private
+    //-------------------------------------------------------------
+
+    @JsonProperty(value = "sub")
+    private String subject;
+    @JsonProperty(value = "aud")
+    private String audience;
+    @JsonProperty(value = "iss")
+    private String issuer;
+    @JsonProperty(value = "token_use")
+    private String tokenUse;
+    @JsonProperty(value = "cognito:username")
+    private String username;
+    private String email;
+    @JsonProperty(value = "email_verified")
+    private boolean emailVerified;
+    @JsonProperty(value = "auth_time")
+    private Long authTime;
+    private String exp;
+    private String iat;
```

Comment:
I'm providing friendly methods to get the parsed ZonedDateTime called `getExpiration` and `getIssuesAt`, that's why I left the raw fields with the same name as the original data